### DISCUSSION TITLE
How do I remove "Source: Article Notebook" from my Quarto manuscript?

### DISCUSSION THREAD

**Discussion by jessicaguo** [quarto-dev/quarto-cli#6999] [Q&A] [answered]
### Description

I was super excited to convert my .Rmd into a quarto manuscript, which was straightforward enough. However, I do have to render to a word doc for my co-authors to comment. When I do, the "Source: Article Notebook" shows up underneath each figure, even though they are static images I created elsewhere in R. How do I turn off this feature?

**Comment by dragonstyle** [comment, marked as answer]
Thanks for jumping into it! 

The idea of linking this way (even for a single computation that contained text + computations) was to allow one copy of the article to be presented as the 'output' and allow linking to a version which contains the source code and outputs.

You can control how the links are presented using the options described here:

https://quarto.org/docs/authoring/notebook-embed.html#linked-source-notebooks

For example, to completely disable, you can use:

```
notebook-links: false
```

HTH and thanks for taking this new capability out for a spin!

**Reply by jessicaguo** [reply]
Thank you all for developing this!

**Reply by jessicaguo** [reply]
Also, its much faster to render my ms in Quarto than Rmd!

**Comment by jzelner** [comment]
Is it possible to specify `notebook-links` on an output-by-output basis? Ideally I'd render to HTML w/links and to pdf w/o for submission and sharing, etc.

Thanks!

-Jon

**Reply by mcanouil** [reply]
What do you mean by "output"?

**Reply by jzelner** [reply]
Sorry - my manuscript outputs to HTML and to PDF via TeX. I'd love to be able to have the PDF w/o notebook links and HTML with them.

**Reply by mcanouil** [reply]
I don't know if it is going to work for this option, but putting it under the format is the usual way to set option specific to a format.

**Comment by jpvelez** [comment]
I have a twist on the original question. I'm using a manuscript, and my `index.qmd` has almost no code, because it pulls in charts and tables from a companion notebook that lives in `/notebooks`, which we are currently linking to globally and inline. So far so good.

The problem is that, due to #8681, our `index.qmd` has to have a tiny amount of code (our goal is to have all compute in the companion notebook, and to have `index.qmd` be text only.) So now there are two notebooks showing up on the sidebar: the companion notebook (good) and the article notebook (bad).

We don't want to show the article notebook because it's confusing. Is there a way to disable the creation of the article notebook, but to show other notebooks? The `notebook-links` option seem to be all of nothing.

**Reply by mcanouil** [reply]
I am not sure it's even possible.

@dragonstyle do you know?

**Reply by mcanouil** [reply]
I've got an idea:

What if you generate a ˋ_variables.yml` file in your notebook and instead of using inline code, you use the variable shortcode to retrieve the values from the file?

**Reply by dragonstyle** [reply]
Does the code that appears inline need to actually output to the document? One workaround is to use:

````
```{r}
# output: false
foo <- "bar"
```
````

As the notebook linking is driven by output, this shouldn't create a link to the notebook when there is no output.

If output is required, you can also exclude a notebook specifically from the `notebooks` treatment using a negation glob, like:

**_quarto.yml**
````
manuscript:
  article: index.qmd
  notebooks:
    - "!index.qmd"
````

which will completely remove the root document from being treated as a notebook by the Manuscript project...

Let us know if this works for you!
